src/functionalTest/resources/features/F-009/S-217.td.json

```diff
@@ -0,0 +1,64 @@
+{
+  "title": "must successfully receive Role Assignments with role name case insensitivity",
+  "_guid_": "S-217",
+  "_extends_": "F-009_Test_Data_Base",
+  "specs": [
+    "contains one role name with upper case chars",
+    "contains other role name with combination of upper and lower case chars"
+  ],
+
+  "request": {
+    "body": {
+      "queryRequests": [
+        {
+          "hasAttributes": ["region"],
+          "roleCategory": ["LEGAL_OPERATIONS"],
+          "roleName": ["TRIBUNAL-CASEWORKER"],
+          "attributes": {
+            "primaryLocation": ["abc"],
+            "jurisdiction": ["IA"]
+          }
+        },
+        {
+          "roleName": ["senior-Tribunal-CASEWORKER"],
+          "attributes": {
+            "primaryLocation": ["abc"],
+            "jurisdiction": ["IA"]
+          }
+        }
+      ]
+    }
+  },
+
+  "expectedResponse": {
+    "_extends_": "Common_200_Response",
+    "body": {
+      "roleAssignmentResponse" : [
+        {
+          "_extends_": "ResponseRoleAssignment",
+          "actorId": "[[ANYTHING_PRESENT]]",
+          "roleCategory" : "LEGAL_OPERATIONS",
+          "roleName" : "[[ANYTHING_PRESENT]]",
+          "attributes" : {
+            "contractType" : "SALARIED",
+            "primaryLocation" : "abc",
+            "jurisdiction" : "IA",
+            "region" : "south-east"
+          }
+        },
+        {
+          "_extends_": "ResponseRoleAssignment",
+          "actorId": "[[ANYTHING_PRESENT]]",
+          "roleCategory" : "LEGAL_OPERATIONS",
+          "roleName" : "[[ANYTHING_PRESENT]]",
```

Comment:
Please provide the explicit role name here to validate the role name case conversion through application.